crates/rocksdb/src/error.rs

```diff
@@ -29,6 +29,9 @@ pub enum RocksError {
     AlreadyOpen,
     #[error(transparent)]
     #[code(unknown)]
+    ExportColumnFamily(rocksdb::Error),
```

Comment:
Not sure if this needs to be its own variant, do you see a place where we need to make the distinction between whether it's an export error vs. any other rocksdb error types?

Reply:
Hm, you're quite right to question this. It didn't seem to fit any of the existing variants but this adds no value over `Other`. Will remove for now.